docs/Indexing_JSON.md

```diff
@@ -53,7 +50,7 @@ In our example we are going to use the following JSON document:
 
 We can use `JSON.SET` to store in our database:
 
-    JSON.SET myDoc $ '{"user":{"name":"John Smith","tag":["foo","bar"],"hp":1000, "dmg":150}}'
+    JSON.SET myDoc $ '{"user":{"name":"John Smith","tag":"foo.bar","hp":1000, "dmg":150}}'
```

Comment:
Should it be a `,`?

Reply:
In my opinion, we better have corresponding tests for our documentation samples so we don't break them, or if we do, we don't have out-of-sync documentation👼🏼